tests/app/service/test_api_key_endpoints.py

```diff
@@ -25,9 +30,12 @@ def test_api_key_should_create_new_api_key_for_service(notify_api, notify_db_ses
             )
             assert response.status_code == 201
             assert 'data' in json.loads(response.get_data(as_text=True))
-            saved_api_key = notify_db_session.session.scalar(select(ApiKey).where(ApiKey.service_id == service.id))
+            saved_api_key: ApiKey = notify_db_session.session.scalar(
+                select(ApiKey).where(ApiKey.service_id == service.id)
+            )
```

Comment:
For consistency with the rest of the code base:
```suggestion
            stmt = select(ApiKey).where(ApiKey.service_id == service.id)
            saved_api_key: ApiKey = notify_db_session.session.scalar(stmt)
```

Reply:
Ah, I missed that one. I meant to convert all these to the proper dao functions.